Slambook/index.js

```diff
@@ -1,18 +1,18 @@
-let form;
-let allSlamArray=[];
-//Write code below to select the form element
-
```

Comment:
The 'getElementsByTagName' method returns a collection, not a single element. If your intention is to select the first form element, you can use 'document.querySelector('form')' instead.